package.json

```
@@ -2,7 +2,7 @@
   "name": "vscode-lizard",
   "displayName": "VS Code Lizard",
   "description": "Run complexity analysis on your code.",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

```suggestion
  "version": "1.0.1",
```

VS Code's publication tool will update this when I publish a new release.